deepspeed/checkpoint/deepspeed_checkpoint.py

```diff
@@ -240,8 +252,8 @@ def _build_transformer_file_map(self):
 
     def _sanity_check(self):
         assert len(self.mp_rank_files) % self.tp_degree == 0
-        assert len(self.layer_keys) > 2
-        assert self.zero_checkpoint.num_files % (self.pp_degree * self.tp_degree) == 0
+        #assert self.zero_checkpoint.num_files % (self.pp_degree * self.tp_degree) == 0
```

Comment:
Should this still be commented out? It seems `self.pp_degree == 1` when there is no pipeline parallelism, right?